simp_le.py

```diff
@@ -1183,5 +1210,33 @@ def test_error_exit_codes(self, dummy_stderr):
                 assert False
 
 
+class IOPluginTests(TestCase):
```

Comment:
This should be `KeyFileTest` and by that I mean both naming and implementation: directly initialize `KeyFile` instead of using `IOPlugin.registered['key.pem']`